packages/jest-each/src/__tests__/array.test.js

```diff
@@ -102,20 +102,20 @@ describe('jest-each', () => {
           ],
         ]);
         const testFunction = get(eachObject, keyPath);
-        testFunction('expected string: %s %d %s %s %d %j %s %j %d %d', noop);
+        testFunction('expected string: %s %d %s %s %d %j %s %j %d %d %_', noop);
 
         const globalMock = get(globalTestMocks, keyPath);
         expect(globalMock).toHaveBeenCalledTimes(2);
         expect(globalMock).toHaveBeenCalledWith(
           `expected string: hello 1 null undefined 1.2 ${JSON.stringify({
             foo: 'bar',
-          })} () => {} [] Infinity NaN`,
+          })} () => {} [] Infinity NaN 0`,
           expectFunction,
         );
         expect(globalMock).toHaveBeenCalledWith(
           `expected string: world 1 null undefined 1.2 ${JSON.stringify({
             baz: 'qux',
-          })} () => {} [] Infinity NaN`,
+          })} () => {} [] Infinity NaN 0`,
```

Comment:
I think this should be 1 as it’s the second row of test data

Reply:
🤦‍♂️